cfme/intelligence/reports/reports.py

```diff
@@ -699,6 +699,9 @@ def step(self, *args, **kwargs):
         )
         self.view.report_info.select()
 
+    def resetter(self, *args, **kwargs):
+        self.obj.browser.refresh()
```

Comment:
Instead of doing a browser refresh in resetter, the recently added `force` kwarg should be used on navigation - this will force re-navigation from the prerequisite, as this is more reliable than a refresh. Browser refreshes can have a significant impact on test execution time.